src/presentation/components/SuccessExamplesLibrary.tsx

```
@@ -0,0 +1,265 @@
+      // Simulate API call
+      await new Promise((resolve) => setTimeout(resolve, 500));
+      setExamples(mockExamples);
+    } catch (error) {
+      console.error("Failed to search examples:", error);
+      setExamples([]);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: searchExamples

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)